src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -47,6 +43,25 @@ public Shooter() {
         subsystemActive = true;
     }
 
+    public double linerToAngular(double linear) {
+        // (velocity(m/s) * 60)/ (2pi * radius)
+        return 0.0;
+    }
+
+    public double calculate(double distance) {
+        double height = CONSTANTS.PORT_HEIGHT - CONSTANTS.HEIGHT_ABOVE_GROUND;
+        double angleRad = Math.toRadians(CONSTANTS.SHOOTER_ANGLE);
+        double velocity = distance / Math.cos(Math.toRadians(angleRad))
```

Comment:
Mind your order of operations here. I think this is not going to do what you expect.

Reply:
(Java will scan from left to right first, and treats * and / with equal precedence)